frontend/packages/dev-console/src/components/monitoring/alerts/monitoring-alerts-utils.tsx

```diff
@@ -58,15 +60,22 @@ export const monitoringAlertColumn: MonitoringAlertColumn[] = [
     fieldName: 'alertState',
     sortFunc: 'alertingRuleStateOrder',
   },
+  { title: 'Notifications', transforms: [cellWidth(20)] },
   { title: '' },
 ];
 
-export const monitoringAlertRows = (alertrules: Rule[], collapsedRowsIds: string[]) => {
+export const monitoringAlertRows = (
+  alertrules: Rule[],
+  collapsedRowsIds: string[],
+  namespace: string,
+) => {
   const rows = [];
   _.forEach(alertrules, (rls) => {
     rows.push({
       ...(rls.state !== RuleStates.Inactive && {
-        isOpen: rls.state === RuleStates.Firing && !_.includes(collapsedRowsIds, rls.name),
+        isOpen:
+          (rls.state === RuleStates.Firing && !_.includes(collapsedRowsIds, rls.name)) ||
```

Comment:
Not introduced by this PR, but it now occurs to me that using the rule `name` as row key will fail if we have two rules with the same `name`.

Reply:
Isn't `alertname` is going to be unique for a namespace?

Reply:
Ah, I forgot this was constrained by namespace.

So, this is the same situation as our silence rule discussion. `alertname` is not technically guaranteed to be unique even within a namespace, but in practice it almost certainly will be. I think this is acceptable for now, but we should probably have a more robust solution at some point.

Reply:
Fixed it, now I am using rules id instead of the name.

Reply:
👍